lisa/tools/git.py

```diff
@@ -201,6 +201,17 @@ def apply(
         )
         result.assert_exit_code(message=f"failed on applying patches. {result.stdout}")
 
+    def am(self, cwd: pathlib.PurePath, patch: pathlib.PurePath) -> None:
```

Comment:
I see you put arg name as "patch" unlike git apply where it has "patches". Does it not work with multiple patches/mboxes? Can you test once if not tested for the tools/library sake?

Reply:
Interesting, I did not notice that.
Though the parameter is named patches, it accepts a Purepath and not list of Purepath.

I will verify the behavior and fix it along with Chi's comment below

Reply:
By multiple I meant something like a pattern.
Ref: https://github.com/microsoft/lisa/blob/5c8d0258a6123094311e1bb7e9d64357fee99ee3/lisa/transformers/kernel_source_installer.py#L75

Reply:
This is different, it is part of the modifier